chart/hyrax/README.md

```diff
@@ -89,6 +89,17 @@ To use an existing or external `fcrepo` instance, use the chart values:
   - `fcrepo.enabled`: false
   - `externalFcrepoHost`: "myfedora.hostname.example.com"
 
+
+# Optional Fits Service
+
+Running fits as a servlet will result in much faster fits responses. This is because the loading cost of the
+various Java components far out weighs the processing time of those components.
+
+To enable fits as a service:
+
+  - `fits.enabled`: true
+  - make sure config/initializers/hyrax.rb has `config.characterization_options = { ch12n_tool: :fits_servlet }`
```

Comment:
i wonder if it's possible to make put this directly into the helm chart? e.g. could `characterization_options` default to this value if the environment variable is set to something remote?

i'm not willing to block on this though. seems okay to just add a note about `config.characterization_options` to whatever the relevant release notes are.

Reply:
https://github.com/samvera/hyrax/pull/6160/commits/20c7ae77810b2a354fcb5ab1d6dc8b1f4ae614b2